docs/manual/manual_safety_systems.rst

```diff
@@ -0,0 +1,174 @@
+
+This means that for any temperature below the range [otw, otlo], there is no derate applied. For any temperature above the range [otw, otlo], the microcontroller 
+temperature derate is 0. Any microcontroller temperature in that range is subject to a variable derate [0, 1].
+
+Suppose you set your otw to 140°C and your otlo to 150°C. This means that there will be no derate limiting applied under 140°C, your module will stop spinning at and above 150°C, and will be derate limited in between.
+
+A graph illustrating the microcontroller temperature derate calculation is provided below:
+
+.. image:: ../_static/manual_images/safety/coil_temp_derate.png
+    :scale: 50%
+
+Calculation of the Total Derate
+-----------------------------------
+The total applied derate applied is the direct multiplication of the speed, microcontroller, and coil derate values. So, the our drive voltage system becomes:
+
+.. image:: ../_static/manual_images/safety/full_derate_system.png
+    :scale: 50%
+
+Power Safety
+=================
+
+Safety Parameters Handled by the Power Safety Client
+-------------------------------------------------------------
+The power safety client manages measurements regarding input power (voltage and current), internal reference power, motor current, microcontroller temperature, and coil temperature.
+
+The full listing of power safety parameters can be found in the :ref:`Power Safety Client table <power_safety>`. Each safety has a low end limit as well as a high end limit. 
+If the measured parameter drops below the low end or goes above the high end, a fault flag is raised, and can be viewed in the ``fault_now`` parameter. 
+The ``fault_now`` parameter is only set by hardware. All faults that have occurred since the module's start up can be viewed in the ``fault_ever`` parameter. 
+The value of ``fault_ever`` is generally set by the hardware and cleared by the user.
+
+The module's behavior when faults are detected is determined by the ``fault_latching`` parameter. If ``fault_latching`` is set to 1, then any high flags 
+in ``fault_ever`` place the module into safe mode. The module remains in safe mode until the user clears ``fault_ever``. When ``fault_latching`` is set to 0, 
+the module remains in safe mode until all power safety parameters are back in their safe operating range, and ``fault_now`` returns to 0.
+
+Internal Programmable Voltage Detector (PVD)
+------------------------------------------------
+Each module's microcontroller has an internal PVD that monitors the microcontroller's Vdd voltage level. In the event that Vdd drops below the pre-programmed threshold 
+value (~2.8V), the module is forced to stop spinning. The module will only be allowed to resume normal operation when Vdd has risen back above the voltage threshold (~2.9V).
+
+.. _slew_rate:
+
+Slew Rate
```

Comment:
As a general terminology thing in this section, I would say that "slew rate" itself is not the thing that does the limiting, slew rate just means rate of change. I would probably call this the "Slew Rate Limiter" and a "slew rate limit" throughout here. Pedantic, I know, but I just like to keep the distinction between the concept of a slew rate and something that limits that slew rate